pression/pluginRegistry.cpp

```diff
@@ -59,8 +59,11 @@ class PluginRegistry
             directories.push_back( getcwd( cwd, MAXPATHLEN ));
 
 #ifdef _WIN32
+            //rgh: does not compile, and seems to be handled by code above.
+            /*
             if( GetModuleFileName( 0, cwd, MAXPATHLEN ) > 0 )
                 directories.push_back( pression::getDirname( cwd ));
```

Comment:
But doesn´t that add it twice? in line 59 and then again in 62? or are both additions needed?
What about the rest of the ifdef? it does not seem to have anything to do with directories, and instead talks about environment variables.